.buildkite/test-vscode-extension.sh

```diff
@@ -10,7 +10,33 @@ apt-get install -y libxshmfence-dev libnss3-dev libatk1.0-0 libatk-bridge2.0-0 \
 
 pushd vscode_extension
 
-yarn
+# https://stackoverflow.com/a/8351489
+with_backoff() {
+  local attempts=5
+  local timeout=1 # doubles each failure
+
+  local attempt=0
+  while true; do
+    attempt=$(( attempt + 1 ))
+    echo "Attempt $attempt"
+    if "$@"; then
+      return 0
+    fi
+
+    if (( attempt >= attempts )); then
```

Comment:
Does this need a `$` prefix, or does the magic of the if make this get evaluated as an arithmetic expression?

Reply:
It works as is (I copied this code directly from test-static-sanitized.sh)

Reply:
I think that in particular the magic of the `(( ... ))` not the `if` makes it work.

Reply:
Also, writing `$((...))` triggers this shellcheck warning:

https://www.shellcheck.net/wiki/SC2084

Reply:
Ah, I see, I was not aware of the difference between [arithmetic expansion](https://www.gnu.org/software/bash/manual/html_node/Arithmetic-Expansion.html#Arithmetic-Expansion) and plain [shell arithmetic](https://www.gnu.org/software/bash/manual/html_node/Shell-Arithmetic.html).  TIL.